pxc/jenkins/qa_pxc_57_80_test-param.yml

```diff
@@ -0,0 +1,62 @@
+- job:
+    name: qa_pxc_57_80_test-param
+    project-type: matrix
+    defaults: global
+    description: |
+        Do not edit this job through the web!
+    disabled: false
+    concurrent: true
+    auth-token: qa_pxc_57_80_param
+    properties:
+    - build-discarder:
+        days-to-keep: -1
+        num-to-keep: 50
+        artifact-days-to-keep: -1
+        artifact-num-to-keep: 50
+    parameters:
+    - string:
+        name: PERCONA_QA_REPO
+        default: "https://github.com/Percona-QA/percona-qa"
+        description: URL to Percona-QA repository
+    - string:
+        name: BRANCH
+        default: "master"
+        description: Tag/Branch for Percona-QA repository
+    - string:
+        name: PXC_LOWER_VERSION_TAR
+        default: "5.7.38-31.59"
+        description: PXC lower version tarball to download for testing
+    - string:
+        name: PXC_UPPER_VERSION_TAR
+        default: "8.0.28-19.1"
+        description: PXC Upper version tarball to download for testing
+    - string:
+        name: PXC57_PKG_VERSION
+        default: "5.7.38-rel41-59.1"
+        description: PXC-5.7 package version
+    axes:
+      - axis:
+         type: user-defined
+         name: node_to_test
+         values:
+             - 'min-centos-7-x64'
+             - 'min-ol-8-x64'
+             - 'min-bionic-x64'
```

Comment:
Please check if we can include Ubuntu 22.04 (Jammy) also

Reply:
Not able to add Ubuntu jammy , it is throwing error : Illegal choice for parameter node_to_test: min-jammy-x64

Reply:
Ok. We can talk to build team later and see how we can resolve this. It is ok for now